CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 **********
 
+4.6.4 - 2025-01-06
+******************
+* Gate use of the Xpert platform v2 endpoint with a waffle flag.
```

Comment:
Nit: would using the v2 endpoint justify considering this a minor release than a patch? So 4.7.0 instead of 4.6.4? Sort of a nit but I wasn't sure how big of a change this would represent api-wise.